lib/foreman_default_hostgroup/engine.rb

```diff
@@ -1,29 +1,29 @@
-require 'default_hostgroup_base_host_patch'
+require "default_hostgroup_base_host_patch"
 
 module ForemanDefaultHostgroup
   class Engine < ::Rails::Engine
-    engine_name 'foreman_default_hostgroup'
+    engine_name "foreman_default_hostgroup"
 
     config.autoload_paths += Dir["#{config.root}/app/models"]
 
-    initializer 'foreman_default_hostgroup.load_default_settings',
+    initializer "foreman_default_hostgroup.load_default_settings",
                 before: :load_config_initializers do
       require_dependency File.expand_path(
-        '../../app/models/setting/default_hostgroup.rb', __dir__
+        "../../app/models/setting/default_hostgroup.rb", __dir__
       )
     end
 
-    initializer 'foreman_default_hostgroup.register_plugin',
+    initializer "foreman_default_hostgroup.register_plugin",
                 before: :finisher_hook do
       Foreman::Plugin.register :foreman_default_hostgroup do
-        requires_foreman '>= 1.17'
+        requires_foreman ">= 2.0"
```

Comment:
should this be 2.2?

Reply:
Agreed @ares.  This should now be updated.